src/main/java/org/broadinstitute/consent/http/models/DataAccessRequestData.java

```diff
@@ -100,7 +100,9 @@ public class DataAccessRequestData {
   private String irbDocumentName;
   private String irbProtocolExpiration;
   private String itDirector;
+  private String itDirectorEmail;
   private String signingOfficial;
+  private String signingOfficialEmail;
```

Comment:
Does this mean that the UI is moving away from sending the `itDirector` and `signingOfficial` fields as part of the payload? Similar question for `piName` and `piEmail`

Reply:
Also, do we need to update `DataAccessRequest.yaml` for the new fields and possibly mark the old ones as deprecated?